refmterr.json

```diff
@@ -12,12 +12,12 @@
     "@reason-native/console": "*"
   },
   "author": {
-    "name": "chenglou"
+    "name": "Reason team at Facebook"
   },
   "devDependencies": {
-    "@esy-ocaml/merlin": "*",
-    "ocaml": "~4.6.0",
-    "@esy-ocaml/reason": "3.3.3"
+    "@opam/merlin": "*",
+    "ocaml": ">= 4.2.0 < 4.9.0",
```

Comment:
We had a tilda range on the dev dependency previously to make sure snapshots didn't break due to small differences in output from the ocaml compiler. I think we still probably want something like that

Reply:
This particular json file isn't used for the tests as far as I recall.